### ISSUE TITLE
@firebase/app as dev dependency causes npm install warnings

### ISSUE THREAD

**Issue by jhuleatt** [firebase/firebase-admin-node#614]
## Describe your environment

  * Operating System version: Mac OS
  * firebase-admin SDK version: 8.3.0

## Describe the problem

Running `npm install` with `firebase-admin` as a dependency results in the following warnings:

```
npm WARN @firebase/database@0.4.11 requires a peer of @firebase/app@0.x but none is installed. You must install peer dependencies yourself.
npm WARN @firebase/database@0.4.11 requires a peer of @firebase/app-types@0.x but none is installed. You must install peer dependencies yourself.
npm WARN @firebase/database-types@0.4.2 requires a peer of @firebase/app-types@0.x but none is installed. You must install peer dependencies yourself.
```

I believe this is caused by this [change in 8.3.0](https://firebase.google.com/support/release-notes/admin/node#version_830_-_july_24_2019):

> Downgraded the dependency on the @firebase/app package to a dev dependency.

## Steps to reproduce:

Clone the functions-samples repo, navigate to the `functions` dir of [big-ben](https://github.com/firebase/functions-samples/tree/master/quickstarts/big-ben), and run `npm install firebase-admin@8.3.0`

**Comment by hiranya911** [comment]
Possible options under consideration:
1. Remove the peer dependency
2. Introduce a new database package specifically for this use case (`@firebase/database-standalone`)

**Comment by Jeremywhiteley** [comment]
I ran to get around it.  
npm install @firebase/app
npm install @firebase/app-types

Then ran install npm install

**Comment by hiranya911** [comment]
Yes, this happened because we downgraded the dependency on `@firebase/app` to a dev dependency. The `@firebase/database` package is designed to work without `@firebase/app`, but it still declares a peer dependency on it. The JS SDK team (@Feiyang1 et al) are already on the case. In the meantime, you're free to ignore the warning. I don't think there are any negative consequences.
